scripts/scripts.js

```diff
@@ -699,14 +699,14 @@ async function loadEager(doc) {
  * loads everything that doesn't need to be delayed.
  */
 async function loadLazy(doc) {
+  await loadHeader(doc.querySelector('header'));
```

Comment:
```suggestion
```

let's move this to `loadEager` instead. We need the header early, so it should go there, instead of us blocking the `loadLazy` function.

Reply:
yeah, I was considering that as well - in the end, I did keep it in loadLazy because that is where it was before but I agree it seems to be more consistent to have in loadEager if we want to mandate it to load first.